### PR TITLE
adding info for configuring vlan support on a pppoe interface.

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

**Review by chao-128t — DISMISSED** [review]

**Review by avinashbhat80 — DISMISSED** [review]

File: docs/howto_pppoe_vlan.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+VLAN support on a PPPoE interface is enabled by configuring the `target-interface` as `<base-eth-interface>.<vlan-id>`.  For example, the intended ethernet interface is dpdk3. If the intended VLAN id is 3, then the target-interface is configured as `dpdk3.3`.
+
+#### Example Configuration:
+
+```
+device-interface pppoe-dev
+    name pppoe-dev
+    type pppoe
+    target-interface dpdk3.3
```
Comment (MichaelBaj) [inline docs/howto_pppoe_vlan.md:14]: where does `target-interface dpdk3.3` come from? Is there a more complete example that can be shown? Also, `dpdk3.3` is not a particularly descriptive name.

Comment (avinashbhat80) [reply docs/howto_pppoe_vlan.md:14]: It is the actual interface name as seen by Linux. The only change this feature requires is to add a `.<vlan>` to that name.

Comment (chao-128t) [reply docs/howto_pppoe_vlan.md:14]: How about?
> For example, if the DSL modem is connected to the Ethernet interface `eth0` on the system and the ISP requires using VLAN tag `1` for the PPPoE connection, the `target-interface` to configure is `eth0.1`.

Comment (MichaelBaj) [reply docs/howto_pppoe_vlan.md:14]: PPPoE interfaces are natively Linux interfaces? We have to presume that a network operator may be coming to this page for the first time without that context. Some language around this detail is important. Also link to existing PPPoE documentation.

Comment (chao-128t) [reply docs/howto_pppoe_vlan.md:14]: Yes, PPPoE interfaces are natively Linux interfaces. 128T uses a set of scripts (running in Linux) to manage the PPPoE interface and leverages KNI, iptable rules, and network namespace to exchange packets with the PPPoE interface.

Comment (MichaelBaj) [reply docs/howto_pppoe_vlan.md:14]: @Chr1st0ph3rTurn3r please incorporate the feedback into this document.

**Review by MichaelBaj — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by chao-128t — APPROVED** [review]

**Pull request merged**